cmd/sys-mgmt-executor/main.go

```diff
@@ -120,26 +128,36 @@ func checkDockerContainerStatus(service string, running bool) (bool, error) {
 	}
 }
 
-func ExecuteDockerCommands(service string, operation string) error {
+func ExecuteDockerCommands(service string, operation string) (string, error) {
 	// don't run commands for unknown services - could be an attack
 	if agent.IsKnownServiceKey(service) {
-		return runDockerCommands(service, operation)
+
+		LoggingClient.Info(fmt.Sprintf("4. IsKnownServiceKey() check a success for service %s", service))
+		LoggingClient.Info(fmt.Sprintf("5. operation: %s", operation))
+
+		_, err := runDockerCommands(service, operation)
```

Comment:
`ExecuteDockerCommands` is effectively a pass-through method which then calls `runDockerCommands`. The check for the known service key can be moved up to `main` and the guts of `runDockerCommands` put into this function to eliminate this empty layer.